src/main/java/org/ideoholic/curium/model/student/service/StudentService.java

```diff
@@ -227,6 +237,80 @@ public boolean viewDetailsOfStudent(String studentId) {
 		return result;
 	}
 
+	public String updateStudent(StudentDto studentDto, MultipartFile[] listOfFiles) {
+		Student student = StudentMapper.INSTANCE.mapStudent(studentDto);
+		Classsec classsec = StudentMapper.INSTANCE.mapClassec(studentDto);
+		Parents parents = StudentMapper.INSTANCE.mapParent(studentDto);
+		Pudetails puDetails = StudentMapper.INSTANCE.mapPudetails(studentDto);
+		Degreedetails degreeDetails = StudentMapper.INSTANCE.mapDegreedetails(studentDto);
+		String id = "";
+		String pid = "";
+		int studentId = 0;
+		int parentsId = 0;
+		String addClass = null, addSec = null, addClassE = null, addSecE = null, conClassStudying = null,
+				conClassAdmittedIn = null;
+		String studentPicUpdate = null;
+		String dropdowncateg = null;
+		String newcateg = null;
+
+		try {
+
+			if (listOfFiles != null && listOfFiles.length != 0) {
+
+				for (MultipartFile fileItem : listOfFiles) {
+					String fileName = (DataUtil.emptyString(fileItem.getOriginalFilename()));
+					String fileValue = (DataUtil.emptyString(fileItem.getName()));
+
+					if (!fileName.equalsIgnoreCase("")) {
+						// Resize the image
+						byte[] bytesEncoded = Base64.encodeBase64(fileItem.getBytes());
+						System.out.println("ecncoded value is " + new String(bytesEncoded));
+						String saveFile = new String(bytesEncoded);
+
+						student.setStudentpic(saveFile);
+
+					} else {
+
+						student.setStudentpic(studentPicUpdate);
```

Comment:
If this field is placed in the mapper then the else condition and this set can be removed
```suggestion
student.setStudentpic(studentPicUpdate);
```